int32 vehicle_id
int32 user_preferred_behavior
int32 user_desired_behavior
float64 user_desired_speed
int32 target_lane